packages/govern-lib/src/thegraph-queries/index.ts

```diff
@@ -0,0 +1,3 @@
+export { QUERY_DAO, QUERY_DAOS } from './dao'
```

Comment:
Btw.: Great to have all those types and queries defined here for the later more decentralized version of Govern! :-)

... I'm not sure if we need here for an internal lib a ``index`` file if we just can import the required queries with an explicit path.